psi4/driver/p4util/fcidump.py

```diff
@@ -229,14 +229,15 @@ def readin_fcidump(fname):
     return header, integrals
 
 
-def compare_fcidumps(expected, computed, digits, label):
-    """Function to compare two FCIDUMP files. Prints :py:func:`util.success`
-    when value *computed* matches value *expected* to the number of *digits*.
+def compare_fcidump_headers(expected, computed, label):
+    """Function to compare the headers of two FCIDUMP files. Prints :py:func:`util.success`
+    when value *computed* matches value *expected*.
     Performs a system exit on failure. Used in input files in the test suite.
 
+    :returns: a tuple of energies computed from the MO integrals (SCF 1-el, SCF 2-el, SCF total, MP2 correlation)
```

Comment:
It might be worth returning a dictionary of energies here. In addition, it would be a neat to have a function that could parse FCIDUMP files to NumPy arrays for toying around separate from the test script.

Reply:
`readin_fcidump` now returns a dictionary with the fields in the header, the core Hamiltonian, the ERIs, the orbital energies and the nuclear repulsion + frozen core energy.